x-pack/plugins/uptime/public/components/common/charts/ping_histogram.test.tsx

```diff
@@ -5,13 +5,23 @@
  */
 
 import React from 'react';
+import DateMath from '@elastic/datemath';
 import { PingHistogramComponent, PingHistogramComponentProps } from './ping_histogram';
-import { renderWithRouter, shallowWithRouter, MountWithReduxProvider } from '../../../lib';
-import moment from 'moment';
+import { render } from '../../../lib';
+import { mockDataPlugin, mockMoment, mockMomentTimezone } from '../../../lib/helper/test_helpers';
 
 describe('PingHistogram component', () => {
+  let dateMathSpy: any;
   beforeAll(() => {
-    moment.prototype.fromNow = jest.fn(() => 'a year ago');
+    mockMoment();
+    mockMomentTimezone();
+    mockDataPlugin();
```

Comment:
instead of running these in each test, do you think we should move these as part of jest setup?

in uptime jest.config.js?

Reply:
We should do this in a follow-up: https://github.com/elastic/uptime/issues/283